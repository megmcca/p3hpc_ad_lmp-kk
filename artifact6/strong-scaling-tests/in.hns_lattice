# Pure HNS crystal, ReaxFF tests for benchmarking LAMMPS
# See README for more info

units		real
atom_style	charge
dimension       3
boundary        p p p
box               tilt large

include hns_lattice.in
region box prism 0 $(v_L*22.326) 0 $(v_L*11.1412) 0 $(v_L*13.778966) 0.0 $(-v_L*5.02603) 0.0 units box
create_box 4 box
include hns_create_atoms.in

set type 1 charge 0.024
set type 2 charge 0.25
set type 3 charge -0.148
set type 4 charge -0.01

pair_style        reaxff NULL
pair_coeff	* * ffield.reax.hns C H O N
mass 1 12
mass 2 1
mass 3 16
mass 4 14



compute   	reax all pair reaxff

neighbor	1.0 bin
neigh_modify      every 20 delay 0 check no

timestep        0.1

thermo_style      custom step temp pe press evdwl ecoul vol spcpu
thermo_modify	norm yes
thermo	  	1000

velocity          all create 300.0 41279 loop geom

fix		1 all nve
fix             2 all qeq/reaxff 1 0.0 10.0 1e-6 reaxff

variable nsteps equal 10000000
fix stop all halt 10 tlimit > 10 error continue
run		${nsteps}
fix stop all halt 10 tlimit > 60 error soft
run		${nsteps}
