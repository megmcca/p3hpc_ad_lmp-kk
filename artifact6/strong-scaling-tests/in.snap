# Demonstrate SNAP W potential

# Initialize simulation

variable a equal 3.1803
units           metal

boundary        p p p

lattice         bcc $a
region          box block 0 $L 0 $L 0 $L
create_box      1 box
create_atoms    1 box

mass 1 183.84

# choose potential

include W_2940_2017_2.snap

# Setup output

thermo          1000
thermo_modify norm yes

# Set up NVE run

timestep 0.5e-3
neighbor 1.0 bin
neigh_modify every 1 delay 0 check yes

# Run MD

velocity all create 800.0 4928459 loop geom
fix 1 all nve

variable nsteps equal 10000000
fix stop all halt 10 tlimit > 10 error continue
run		${nsteps}
fix stop all halt 10 tlimit > 60 error soft
run		${nsteps}
